daffodil-lib/src/main/scala/org/apache/daffodil/lib/cookers/Cookers.scala

```diff
@@ -101,8 +101,6 @@ object ChoiceDispatchKeyCooker extends StringLiteralNonEmptyNoCharClassEntitiesN
 object ChoiceBranchKeyCooker
   extends ListOfStringLiteralNonEmptyNoCharClassEntitiesNoByteEntities()
 
-object RepValueCooker extends ListOfStringLiteralNonEmptyNoCharClassEntitiesNoByteEntities()
```

Comment:
If this is deleted, is that because repValues and range end points can only be integers?

Reply:
Follow up. I've never seen it, but are our repValues integers nonNegative only, or can things like -1 -2, etc. be enum repValues?

Reply:
I think I can actually bring back the cooker. I thought this cooked everything to a BigInt and so avoided it (one of the updates in the PR is to use the correct DataPrimitive type). But I now see it just cooks to a list of strings. This is better than the `.split` stuff I do elsewhere. I'll give it a shot.

Reply:
> Follow up. I've never seen it, but are our repValues integers nonNegative only, or can things like -1 -2, etc. be enum repValues?

The code currently allows anything derived from xs:integer, so signed types should work as well. I'll add a test or two to make sure it works as expected.